cmd/nerdctl/push.go

```diff
@@ -68,6 +69,7 @@ func newPushCommand() *cobra.Command {
 
 	pushCommand.Flags().Bool("estargz", false, "Convert the image into eStargz")
 	pushCommand.Flags().Bool("ipfs-ensure-image", true, "Ensure the entire contents of the image is locally available before push")
+	pushCommand.Flags().String("ipfs-address", "", "multiaddr of IPFS API (default uses $IPFS_PATH env variable if defined or local directory ~/.ipfs)")
```

Comment:
Could you add this flag to the Command Reference in `README.md` too?

Reply:
done